backend/app/store/engine/engine.go

```diff
@@ -73,11 +74,12 @@ const (
 	FlagFalse  FlagStatus = -1
 )
 
-// Enum of all flags
+// Enum of all flags and details
 const (
 	ReadOnly = Flag("readonly")
 	Verified = Flag("verified")
 	Blocked  = Flag("blocked")
+	Email    = UserDetail("email")
```

Comment:
looks like an alien here. You are preparing to have different flavours of UserDetails, so don't mix itto another family but make it's own const group (with one for now)